scripts/ci/preflight-steps/dependency_links.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
```

Comment:
I'm going to propose that we don't duplicate this information, see [this](https://github.com/model-checking/kani/pull/2314/files#r1146533803)